docs/tutorials/notebooks/basic_ot_between_datasets.ipynb

```diff
@@ -41,13 +41,18 @@
    "metadata": {},
```

Comment:
<p>Please link to <code>Sinkhorn</code> as {func}<code>~ott.solvers.linear.sinkhorn.Sinkhorn</code> </p><p>Also add {func}<code>~ott.utils.default_progress_fn</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>